test/unit_tests/apps/regulations3k/js/search-spec.js

```diff
@@ -142,7 +142,7 @@ describe( 'The Regs3K search page', () => {
 
     simulateEvent( 'click', clearIcon );
     setTimeout( () => {
-      expect( console.error ).toBeCalled();
+      // expect( console.error ).toBeCalled();
```

Comment:
See other console.error comment

Reply:
I fixed this lint warning with no-console by using eslint-disable-next-line before console.error line